solana/program_v2/packages/client/core/src/GatekeeperService.ts

```diff
@@ -245,6 +247,7 @@ export class GatekeeperService extends AbstractService {
         networkTokenAccount: networkTokenAccount,
         gatekeeperTokenAccount: gatekeeperTokenAccount,
         funderTokenAccount: funderTokenAccount,
+        feePayer: funderAuthority,
```

Comment:
resolve naming inconsistency. I'd propose rentPayer, but this should then be consistent across all interfaces.

Reply:
This is the key paying for the passes, think funder makes sense? Since we have a funder token account.